builtin.c

```diff
@@ -474,6 +474,19 @@ XS(XS_builtin_indexed)
     XSRETURN(retcount);
 }
 
+XS(XS_builtin_load_module);
+XS(XS_builtin_load_module)
+{
+    dXSARGS;
+    if (items != 1)
+        croak_xs_usage(cv, "arg");
+    SV *module_name = newSVsv(ST(0));
+    if (!SvPOK(module_name))
+        croak_xs_usage(cv, "defined string");
```

Comment:
The call to load_module() will release `module_name`, but `module_name` isn't released if the croak is called.

Reply:
Thanks! I've added a `SvREFCNT_dec(module_name);` prior to calling `croak_xs_usage(cv, "defined string");`. That should do it, right?